### PR TITLE
53 version 9 0

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@Yuri05 Update to version 9.0, vc++ 2019 (I could not find a fwlink for that) and dotnet472

**Comment by msevestre** [comment]
@Yuri05 This still point to the old R package implementation. I feel that we should not package this anymore with the suite

**Comment by Yuri05** [comment]
>I feel that we should not package this anymore with the suite

What does this mean? Currently a ZIP archived package ist just copied to users drive during the install.
You suggest to replace it with download on demand?

**Comment by msevestre** [comment]
@Yuri05 Yes. I do not understand why we think this is a good idea. Woudn't it be better to have a link to github to downlaod the package ? That way it is the same for everyone, even people using the portable version etc.

**Comment by Yuri05** [comment]
@msevestre OK, let's remove R-toolbox (zip archiv) from the OSP release.

File: rakefile.rb [hunk 1]
```diff
@@ -31,8 +31,6 @@
 
   MSI[:pksim] = create_package('pk-sim', 'PK-Sim')
   MSI[:mobi] = create_package('mobi', 'MoBi')
-  MSI[:matlab] = create_package('matlab-toolbox', 'Matlab-Toolbox')
-  MSI[:r] = create_package('r-toolbox', 'R-Toolbox')
```
Comment (msevestre) [inline rakefile.rb]: Gone!

**Pull request merged**